<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mywindow</class>
 <widget class="QMainWindow" name="mywindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>400</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="baseSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>카페 이메일 추출기</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton1">
    <property name="geometry">
     <rect>
      <x>429</x>
      <y>297</y>
      <width>75</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>추출 시작</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>437</x>
      <y>32</y>
      <width>151</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>26</pointsize>
     </font>
    </property>
    <property name="text">
     <string>사용 방법</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>405</x>
      <y>127</y>
      <width>231</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>1. 원하시는 카페 이름을 입력해주세요.</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>418</x>
      <y>146</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>예) http://cafe.naver.com/darias</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>280</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton2">
    <property name="geometry">
     <rect>
      <x>513</x>
      <y>297</y>
      <width>75</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>작업 종료</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radio1">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>246</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>이메일 형식(메일 발송용)</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radio2">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>267</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>아이디 형식(쪽지 발송용)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>432</x>
      <y>188</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>405</x>
      <y>223</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>2. 원하시는 저장 방식을 지정해주세요.</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>93</x>
      <y>43</y>
      <width>211</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>프로그램을 이용해주셔서 감사합니다.</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>63</y>
      <width>361</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>IP 일시차단이 발생할 수 있으므로 추출 후 10분은 휴식해주세요.</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>31</x>
      <y>100</y>
      <width>331</width>
      <height>251</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string/>
     </property>
     <property name="font">
      <font>
       <family>돋움</family>
       <pointsize>9</pointsize>
      </font>
     </property>
    </item>
    <item>
     <property name="text">
      <string/>
     </property>
    </item>
    <item>
     <property name="text">
      <string/>
     </property>
    </item>
    <item>
     <property name="text">
      <string/>
     </property>
     <property name="font">
      <font>
       <family>돋움</family>
      </font>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </item>
    <item>
     <property name="text">
      <string/>
     </property>
     <property name="font">
      <font>
       <family>돋움</family>
      </font>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </item>
    <item>
     <property name="text">
      <string/>
     </property>
    </item>
    <item>
     <property name="text">
      <string/>
     </property>
     <property name="font">
      <font>
       <family>돋움</family>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </item>
    <item>
     <property name="text">
      <string/>
     </property>
    </item>
    <item>
     <property name="text">
      <string/>
     </property>
    </item>
    <item>
     <property name="text">
      <string/>
     </property>
     <property name="font">
      <font>
       <family>돋움</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </item>
    <item>
     <property name="text">
      <string/>
     </property>
    </item>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>388</x>
      <y>100</y>
      <width>241</width>
      <height>251</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>46</x>
       <y>67</y>
       <width>151</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>아래 빈칸에'darias'만 입력</string>
     </property>
    </widget>
   </widget>
   <zorder>groupBox</zorder>
   <zorder>pushButton1</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>label_3</zorder>
   <zorder>label_4</zorder>
   <zorder>pushButton2</zorder>
   <zorder>radio1</zorder>
   <zorder>radio2</zorder>
   <zorder>lineEdit</zorder>
   <zorder>label_5</zorder>
   <zorder>label_6</zorder>
   <zorder>label_7</zorder>
   <zorder>listWidget</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
